src/main/java/org/folio/linked/data/service/impl/ResourceServiceImpl.java

```diff
@@ -117,15 +118,24 @@ public Resource saveMergingGraph(Resource resource) {
 
   private Resource saveMergingGraphSkippingAlreadySaved(Resource resource, Resource saved) {
     if (resource.isNew()) {
-      if (doesNotExists(resource)) {
-        resourceRepo.save(resource);
-      }
```

Comment:
@askhat-abishev, @pkjacob, after this check the bug https://folio-org.atlassian.net/browse/MODLD-445 will probably gone, because we refresh all incoming resources here, not only not existed